dandischema/digests/zarr.py

```diff
@@ -0,0 +1,148 @@
+from __future__ import annotations
+
+import hashlib
+from typing import Dict, List, Optional
+
+import pydantic
+
+
+"""Passed to the json() method of pydantic models for serialization."""
+ENCODING_KWARGS = {"separators": (",", ":")}
+
+
+class ZarrChecksum(pydantic.BaseModel):
+    """
+    A checksum for a single file/directory in a zarr file.
+
+    Every file and directory in a zarr archive has a path and a MD5 hash.
+    """
+
+    md5: str
+    path: str
+
+    # To make ZarrChecksums sortable
+    def __lt__(self, other: ZarrChecksum):
```

Comment:
You should decorate the class with [`@functools.total_ordering`](https://docs.python.org/3/library/functools.html#functools.total_ordering) for the remaining comparison operators.